server/src/main/java/org/opensearch/extensions/ExtensionsManager.java

```diff
@@ -35,12 +35,14 @@
 import org.opensearch.cluster.ClusterSettingsResponse;
 import org.opensearch.cluster.node.DiscoveryNode;
 import org.opensearch.cluster.service.ClusterService;
+import org.opensearch.common.ParsingException;
```

Comment:
I know I mentioned "ParsingException" in my description, but it was meant generally. I didn't realize we actually had one.   

I think an IOException is fine to throw here so we don't need to add a separate exception type.

Reply:
Done